src/app/main/service/localstorage/local-storage.service.ts

```diff
@@ -23,6 +24,14 @@ export class LocalStorageService {
   accessTokenBehaviourSubject: BehaviorSubject<string> = new BehaviorSubject<string>(this.getAccessToken());
   ubsRegBehaviourSubject: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(this.getUbsRegistration());
 
+  public setHabitsGalleryView(value: boolean): void {
+    localStorage.setItem(this.HABITS_GALLERY_VIEW, `${value}`);
+  }
+
+  public getHabitsGalleryView(): string | null {
+    return localStorage.getItem(this.HABITS_GALLERY_VIEW);
+  }
```

Comment:
```suggestion
  public getHabitsGalleryView(): boolean | null {
    return JSON.parse(localStorage.getItem(this.HABITS_GALLERY_VIEW));
  }
```
It is better to return it as boolean to keep contracts more straightforward.